Common/Source/Dialogs/dlgWayQuick.cpp

```diff
@@ -125,7 +125,35 @@ static void OnRadioFrequencyClicked(WndButton* pWnd){
 
   devPutFreqActive(Ferquency, WayPointList[SelectedWaypoint].Name);
 
-  _stprintf(szFreq,_T(" %6.3f ") ,Ferquency);
+  _stprintf(szFreq,_T("Active %6.3f ") ,Ferquency);
+
+  DoStatusMessage(_T(""), WayPointList[SelectedWaypoint].Name );
+  DoStatusMessage(_T(""), szFreq );
+  retStatus=3;
+  if(pWnd) {
+    WndForm * pForm = pWnd->GetParentWndForm();
+    if(pForm) {
+      pForm->SetModalResult(mrOK);
+    }
+  }
+#endif  // RADIO_ACTIVE
+}
+
+
+
+static void OnRadioFrequencySBClicked(WndButton* pWnd){
```

Comment:
OnRadioFrequencySBClicked & OnRadioFrequencyClicked are almost the same (only 2 line of 25 have few difference), must be merged ...